testsuite/python/lb_momentum_conservation.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2010-2018 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import espressomd
+from espressomd import lb, lbboundaries, shapes, has_features
+import unittest as ut
+import numpy as np
+import sys
+
+# Define the LB Parameters
+TIME_STEP = 0.01
+AGRID = 1.0 
```

Comment:
please change the value of ```AGRID``` to some value != 1.0

Reply:
It is now 0.4.